build/metadata.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+# coding: utf-8
+"""
+Process manuscript metadata and author metadata
+"""
+import collections
+import pathlib
+
+import yaml
+
+
+def get_metadata_object(path):
+    """
+    Load all of the metadata from a yaml file, & return.
+
+    :param path: pathlib.Path to a metadata YAML file.
+    :return: structured metadata object.
+    """
+    with path.open('r') as read_file:
+        metadata_dict = yaml.load(read_file)
+    initials = []
+    metadata_dict['author'] = []
+    # Check for duplicated initials
+    for author in metadata_dict['author_info']:
+        initials.append(author['initials'])
+        metadata_dict['author'].append(author['full_name'])
+    initials_counts = collections.Counter(initials)
+    if max(initials_counts.values()) > 1:
+        duplicated_initials = [k for k, v in d.items() if v > 1]
+        msg = f'Duplicated initials in metadata.yaml: {duplicated_initials}'
+        raise ValueError(msg)
+    metadata = dict()
+    metadata['author_info'] = metadata_dict.pop('author_info')
+    metadata['header_block'] = metadata_dict
```

Comment:
In any case, the list of full names to go in `pandoc_metadata['author']` will be created in Python from  `metadata['author_info']['full_name']`. I don't think it will cause an issue to pass extra metadata to pandoc... especially since oftentimes that will be a feature. The question is do we know now all the metadata fields we want to passthrough? I don't think we do, so it may be best to passthrough everything and just make sure not to define fields that are used by pandoc unless that is the intention. For example, pandoc-tablenos has metadata options like `xnos-section-numbers: On`. This we would want to passthrough.

> Lastly, we may want to decouple the validation of the metadata (e.g., title) and the validation of the author information (e.g., names) to allow for double blind peer-review if necessary.

Double blind peer review is not supported by this system.